src/main/java/com/sampleweb/pages/PCVideoGamesPage.java

```
@@ -0,0 +1,219 @@
+  private WebElement firstNavLink;
+
+  @FindBy(xpath = "//input[@type='checkbox' and @name='s-ref-checkbox-419158031']")
+  private WebElement primeCheckBox;
+
+  @FindBy(xpath = "//div[contains(@class, 's-search-results')]/div")
+  private List<WebElement> products;
+
+  @FindBy(xpath = "//input[@id='twotabsearchtextbox']")
+  private WebElement searchBox;
+
+  @FindBy(xpath = "//span[text() = '4 Stars & Up']")
+  private WebElement fourStarReview;
+
+  @FindBy(xpath = "//select[@id='s-result-sort-select']/option[text()='Avg. Customer Review']")
+  private WebElement avgReviewSortOption;
+
+
+  public PCVideoGamesPage(RemoteWebDriver driver) {
+    super(driver);
+  }
+
+  public boolean isLoaded() {
+    return title.isDisplayed();
+  }
+
+  public void navigateByDropdown() {
+    // Get the dropdown and select the video games item
+    Select drop = new Select(dropDown);
+    drop.selectByValue("search-alias=videogames");
+    // Empty the search box and then search
+    searchBox.clear();
+    submitButtonLocator.click();
+  }
+
+  public boolean hasGlobal() {
+    return globalStore.isDisplayed();
+  }
+
+  public boolean hasAvgReview() {
+    return avgReviewHeader.isDisplayed();
+  }
+
+  public boolean hasSameFirstNavLink() {
+    return firstNavLink.isDisplayed();
+  }
+
+  public boolean firstLinkSameNav() {
+    firstNavLink.click();
+    return isLoaded();
+  }
+
+  public boolean checkPrimeItems() {
+    // Check the prime checkbox
+    primeCheckBox.click();
+
+    // All the products
+    for (WebElement product : products) {
+      try {
+        // contains the prime label or a span which contains more buying choices
+        // don't need to store in a variable, just want it too exist
+        product.findElement(By.xpath(
+            ".//i[@aria-label='Amazon Prime'] | .//span[contains(text(), 'More buying choices')]"))
+               .isDisplayed();
+      } catch (NoSuchElementException e) {
```

No need to catch the exception here. Just call the '.findElement' and let the test fail if an exception is raised.